### PR TITLE
Add volume toggle docs

### PR DESCRIPTION
**Pull request by jameslkingsley**
**When merged this pull request will:**
- Title

### CODE REVIEW

**Comment by bux** [comment]
Is there a default keybind defined by us? Then this should go into that doc here.

**Comment by Dystopian** [comment]
There isn't atm. I wish there was a default keybind like in Cyprus Push-to-Hear (Right Windows).

File: docs/wiki/featurex/volume.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  patch: 0
+---
+
+## 1. Overview
+
+Adds a keybind to toggle volume of game and music. It's quite useful for when you're briefing in heavy rain, or if you're in an obnoxiously loud car.
+
+## 2. Dependencies
+
+{% include dependenciesx_list.md component="volume" %}
```
Comment (jonpas) [inline docs/wiki/featurex/volume.md]: Dependencies section last.

File: docs/wiki/featurex/volume.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## 3. Features
+* Automatically lower volume when inside vehicles (restores when exiting vehicles)
```
Comment (jonpas) [inline docs/wiki/featurex/volume.md]: Use `-` instead of `*`.

File: docs/wiki/featurex/volume.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## 1. Overview
+
+Adds a keybind to toggle volume of game and music. It's quite useful for when you're briefing in heavy rain, or if you're in an obnoxiously loud car.
+
+## 2. Dependencies
+
+{% include dependenciesx_list.md component="volume" %}
+
+## 3. Features
```
Comment (jonpas) [inline docs/wiki/featurex/volume.md]: Empty line required after headers.

File: docs/wiki/featurex/volume.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+mod: acex
+version:
+  major: 3
+  minor: 2
+  patch: 0
+---
+
+## 1. Overview
+
+Adds a keybind to toggle volume of game and music. It's quite useful for when you're briefing in heavy rain, or if you're in an obnoxiously loud car.
```
Comment (jonpas) [inline docs/wiki/featurex/volume.md]: Maybe write the part about the keybind differently, eg. "Adds the possibility of setting a keybind to toggle ..."?

File: docs/wiki/featurex/volume.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
+layout: wiki
+title: Volume Toggle
+description: Keybind to toggle volume of game and music.
```
Comment (jonpas) [inline docs/wiki/featurex/volume.md]: "Ability to toggle ..." I'd say.

**Pull request merged**